cmd/helmhook/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright (C) 2022-2024 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/pflag"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/apecloud/kubeblocks/cmd/helmhook/hook"
+	_ "github.com/apecloud/kubeblocks/cmd/helmhook/hook/multiversion"
+)
+
+var (
+	crdPath    string
+	version    string
+	namespace  string
+	keepAddons bool
+)
+
+func setupFlags() {
+	pflag.StringVar(&crdPath, "crd", "/kubeblocks/crd", "CRD directory for the kubeblocks")
+	pflag.StringVar(&version, "version", "", "KubeBlocks version")
+	pflag.StringVar(&namespace, "namespace", "default", "The namespace scope for this request")
+	pflag.BoolVar(&keepAddons, "keep-addons", true, "Whether to allow addon updates")
```

Comment:
Whether to allow addon updates, if true, don't.

Reply:
updated